spec/unit/blitbuffer_spec.lua

```diff
@@ -31,6 +50,191 @@ describe("Blitbuffer unit tests", function()
             local c4u_32 = cRGB32:getColor4U()
             assert.are.equals(0xB0, c4u_32.a)
         end)
+
+        it("should have pure white stay pure white when converting", function()
+            print("Converting from RGB32")
```

Comment:
I think it would be slightly nicer to write that as follows, don't you? :-)

```lua
describe("should have pure white stay pure white when converting", function()
    it("from RGB32"), function()
        -- etc
```

Reply:
PS Very nice!

Reply:
I do (this unit test was obviously dedicated to you, as you would have requested one :) - and seeing if other conversions need a fix.)
(Can't really test the whole describe/it stuff, so hoping for no typo.)

Reply:
Does the white dot (instead of green dot) on a unique line means anything?
Blitbuffer unit tests Color conversion should have pure white stay pure white when converting from RGB8 ... ●

Reply:
OK, no more white dot on last travis run.